src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2207,6 +2210,118 @@
     crafted: true
 """
 
+helps['webapp sitecontainers create'] = """
+type: command
+short-summary: Create sitecontainers for a linux webapp
+long-summary: |
+    Multiple sitecontainers can be added at once by passing arg --sitecontainer-spec-file, which is the path to a json file containing an array of sitecontainer specs.
+    Example json file:
```

Comment:
How about the json to have these two items at top:
{
name:
properties: {ismain: , target_port: , ...}
}
Sidecar name is not part of properties and this format will look for like the ARM spec and matching will be easier.